admin/templates/nodes/node.html

```diff
@@ -416,6 +429,42 @@ <h3>Are you sure you want to update this embargo?</h3>
                     <pre>{{ node.spam_data }}</pre>
                 </td>
             </tr>
+            <tr>
+                <td>Node storage usage</td>
+                <td>
+                    <b>Current usage:</b> {{ node.storage_usage }}<br>
+                    <b>Public cap:</b> {{ node.public_storage_cap }} GB<br>
+                    <b>Private cap:</b> {{ node.private_storage_cap}} GB<br>
+                    <b>Storage usage status:</b> {% include "nodes/storage_status.html" with resource=node %}
+                </td>
+                <td>
+                    <a href="{% url 'nodes:adjust-storage-usage' guid=node.id %}"
```

Comment:
Kinda curious why the digits of precision seem to be different for public and private caps, maybe it's something about my data, but obviously kind of weird. Pic for context.

<img width="789" alt="Screen Shot 2020-09-18 at 11 06 44 AM" src="https://user-images.githubusercontent.com/9688518/93613671-2c812980-f99f-11ea-94ca-73f3f2026863.png">

also super nit: the input boxes aren't aligned vertically 100% and there's no styling on the save button.

Reply:
Yeah this was kind of weird. For some reason the defaults just show '5' or '50', but the custom storage caps show all the decimals. I will try to put in formatting to prevent this.